src/seedu/addressbook/data/person/Person.java

```diff
@@ -13,13 +13,13 @@
     private Name name;
     private Phone phone;
     private Email email;
-    private Address address;
+    private Contact address;
 
     private final UniqueTagList tags;
     /**
      * Assumption: Every field must be present and not null.
      */
-    public Person(Name name, Phone phone, Email email, Address address, UniqueTagList tags) {
+    public Person(Name name, Phone phone, Email email, Contact address, UniqueTagList tags) {
```

Comment:
I don't understand the requirement for this change. Shouldn't it remain as Address?